api/server.py

```diff
@@ -166,4 +179,70 @@ def expression():
         body = request.json
         response = database_lib.expression(body).to_list()
         return jsonify(response)
-        
\ No newline at end of file
+
+@app.route('/api/content.json', methods=['POST', 'GET'])
+def content():
```

Comment:
Is it an issue losing the POST if the user can't be verified or does that not matter?

Reply:
It won't matter. The frontend would never suggest to POST in this case :heavy_check_mark: